.github/workflows/build_and_test.yml

```diff
@@ -1,44 +1,82 @@
-name: build-and-test
-on: [push]
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-      - name: Build and export
-        uses: docker/build-push-action@v3
-        with:
-          context: .
-          file: ./Dockerfile
-          tags: meals-api:latest
-          outputs: type=docker,dest=/tmp/image.tar
-
-      - name: Write build result to file
-        if: always()
-        run: |
-          datetime=$(date +"%Y-%m-%d %H:%M:%S")
-          names="Daniel, Omer, Shachar"
-          echo $datetime >> build_result.txt
-          echo $names >> build_result.txt
-          if [ -f /tmp/image.tar ]; then
-            echo "Image successfully built" >> build_result.txt
-          else
-            echo "Image not able to be built" >> build_result.txt
-          fi
-
-      - name: Upload image artifact
-        uses: actions/upload-artifact@v3
-        with:
-          name: meals-api
-          path: /tmp/image.tar
-
-      - name: Upload result file
-        if: always()
-        uses: actions/upload-artifact@v3
-        with:
-          name: meals-api-build-result.txt
-          path: build_result.txt
\ No newline at end of file
+name: build-and-test
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and export
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: ./Dockerfile
+          tags: meals-api:latest
+          outputs: type=docker,dest=/tmp/image.tar
+
+      - name: Write build result to file
+        if: always()
+        run: |
+          datetime=$(date +"%Y-%m-%d %H:%M:%S")
+          names="Daniel, Omer, Shachar"
+          echo $datetime >> build_result.txt
+          echo $names >> build_result.txt
+          if [ -f /tmp/image.tar ]; then
+            echo "Image successfully built" >> build_result.txt
+          else
+            echo "Image not able to be built" >> build_result.txt
+          fi
+
+      - name: Upload image artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: meals-api
+          path: /tmp/image.tar
+
+      - name: Upload result file
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: meals-api-build-result.txt

```

Comment:
can you please rename this with `log.txt`? It's the requirement, just I didn't read it properly when I did this.